openfisca_france/assets/taxe_habitation/source/do_REI2017_to_csv.do

```diff
@@ -0,0 +1,48 @@
+/***************************************************************************************/
```

Comment:
I'm not sure this file should be commited in this repository

Reply:
Ok pour le conserver mais ce serait alors dans `openfisca-france/openfisca_france/scripts`.
Et dans ce cas, pour le rendre utilisable, peux-tu indiquer la version de stata employée (aka, le nécessaire à avoir dans l'environnement pour l'appeler, librairies...) ainsi que la commande pour l'exécuter ?

Reply:
En as-tu une version en Python pour accroître ses chances d'être mis à jour ?

Reply:
@sandcha : j'ai ajouté la version de stata nécessaire pour lancer le programme. Pour l'emplacement, je trouve qu'il est mieux de garder le programme près du fichier .csv sinon, peut-être qu'il n'est pas évident de savoir que ce programme existe lorsqu'on consulte le csv.